arlas-rest/src/main/java/io/arlas/server/rest/explore/ExploreRESTServices.java

```diff
@@ -50,11 +51,13 @@ public ExploreServices getExploreServices() {
     }
 
     protected ExploreServices exploreServices;
+    protected ElasticAdmin elasticAdmin;
 
```

Comment:
ElasticAdmin should be isolated from RESTServices and ParamsParser

Reply:
Fixed